processor/src/main/java/com/workday/autoparse/xml/codegen/XmlElementParserGenerator.java

```diff
@@ -81,6 +84,14 @@ public void generateParser()
         writer.emitImports(getStandardImports());
         writer.emitEmptyLine();
 
+        final XmlElement xmlElementAnnotation = classElement.getAnnotation(XmlElement.class);
```

Comment:
This is where it becomes incremental?

Reply:
no this only helps kick off the second processor. 

this makes it incremental
processor/src/main/resources/META-INF/gradle/incremental.annotation.processors
and the fact that we separated the processors and added originating elements

```
 JavaFileObject sourceFile = processingEnv.getFiler().createSourceFile(parserName, classElement);
```